Data/Sys/GameSettings/GLC.ini

```diff
@@ -21,7 +21,6 @@ EmulationStateId = 4
 
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
-DstAlphaPass = True
```

Comment:
Ah, Lemony Snicket gameini was made by Skidau if i remember correctly. I don't know why it is there really.